packages/@webex/internal-plugin-metrics/src/metrics.types.ts

```diff
@@ -200,9 +211,8 @@ export type SubmitClientEvent = (args: {
 }) => Promise<any>;
 
 export type SubmitOperationalEvent = (args: {
-  name: OperationalEvent['name'];
-  payload?: RecursivePartial<OperationalEvent['payload']>;
-  options?: any;
+  name: OperationalEvent['metricName'];
```

Comment:
could we add a constants file for all these accessors, such as `metricName`, 'payload', etc? for ease of use and lower chance of typos

Reply:
Actually I was wondering about those: I saw this pattern in previous code I change so I didn't want to change it, but what's wrong with `string`? (I get the point of using this syntax for complex types/subject to changes though). 

Otherwise, sure, I can move those under a constant... Or isn't the following a better pattern? (It's an actual question - I don't have much experience in this code base/TS):

```
type MetricName = string;

type OperationalEvent {
....
 metricName: MetricName
```

Reply:
no because then they are not linked together, i.e., OperationalEvent['metricName'] type may change at some point in time from string to enum, or something along these lines

Reply:
Sure yeah. I don't see any example of this in metric code yet (looking at other metrics). Should I add a simple

```
const MetricNameKey = "metricName"
```

then replace all occurrences of `OperationalEvent['metricName']` into `OperationalEvent[MetricNameKey]` then? (similarly for behavioral and business of course).

Reply:
yes

Reply:
Is this addressed? I do not see this change happened yet?

Reply:
No - after trying it it turns out that putting a symbol (variable) instead of a literal string changes the semantic and break the build. Discussed with @chrisadubois we'll leave as it is.